docs/modules/configuring/con-configuring-vc-listeners.adoc

```diff
@@ -0,0 +1,276 @@
+[id='con-configuring-vc-listeners-{context}']
+= Configuring virtual cluster listeners
+
+[role="_abstract"]
+
+Clients make connections to virtual cluster listeners. Each listener defines a bootstrap address.  This is the address
+that the client makes its initial connection to. The listener also provides a means for the clients to talk to the
+proxied brokers. There is a choice about how this is done:
+
+* *Port Identifies Node* - the listener binds separate ports - one for each broker as well as a separate one for bootstrap.
+  Clients make connections to the different _port numbers_ as they need to converse with each broker.
+* *SNI Host Identifies Node* - the listener assigns a distinct hostname to each broker.
+  Clients make connections to the different _hostnames_ as they need to converse with each broker.
+  The listener uses SNI (Server Name Identification) to work out which broker the client wishes to talk to
```

Comment:
I like the change in terminology to `X Identifies Node` a lot clearer than anything we have had previously.